src/core/core.tooltip.js

```diff
@@ -530,10 +530,14 @@ var exports = Element.extend({
 			var bodyItem = {
 				before: [],
 				lines: [],
+				firstLine: [],
```

Comment:
instead of creating this `firstLine` array I would change the `helpers.each` call to a standard `for` loop and check if `i === 0`